src/orders/components/OrderChangeWarehouseDialog/styles.ts

```diff
@@ -38,6 +38,12 @@ export const useStyles = makeStyles(
       fontSize: "14px",
       lineHeight: "160%",
       color: theme.palette.saleor.main[3]
+    },
+    loader: {
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      padding: theme.spacing(2)
     }
```

Comment:
It's not a change request, but I think in the future we may want to think about making a global style/component for this, it's quite common loader wrapper in the dashboard, search for `loadMoreLoaderContainer` or `progressContainer`

Reply:
Circular Progress is used in too many components to include this refactor in this PR, but it's a good idea.

Reply:
Please check my comment above as probably loading screens should be done differently